modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -457,8 +457,9 @@ public void setFileIOFactory(FileIOFactory ioFactory) {
                         for (File f : walWorkDir0.listFiles())
                             size += f.length();
 
-                        for (File f : walArchiveDir0.listFiles())
-                            size += f.length();
+                        if (!walWorkDir0.equals(walArchiveDir0))
```

Comment:
It is more correct to use the method:
org.apache.ignite.internal.processors.cache.persistence.wal.FileWriteAheadLogManager#isArchiverEnabled

Don't forget about curly braces.